nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -160,14 +160,17 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The `erigon` ethereum node has moved it's database location in `2021-08-03`, users upgrading must manually move their chaindata (see [release notes](https://github.com/ledgerwatch/erigon/releases/tag/v2021.08.03)).
 
 - [users.users.&lt;name&gt;.group](options.html#opt-users.users._name_.group) no longer defaults to `nogroup`, which was insecure. Out-of-tree modules are likely to require adaptation: instead of
+
   ```nix
   {
     users.users.foo = {
       isSystemUser = true;
     };
   }
   ```
+
   also create a group for your user:
+
```

Comment:
These changes are unrelated to your PR and should not be included in its commit

Reply:
Oops, good catch! Fixing it now.